src/Uno.UI/DataBinding/BindingPath.cs

```diff
@@ -827,14 +818,6 @@ private void BuildValueGetter()
 				}
 			}
 
-			private void BuildPrecedenceSpecificValueGetter()
-			{
-				if (_precedenceSpecificGetter == null && _dataContextType != null)
-				{
-					_precedenceSpecificGetter = BindingPropertyHelper.GetValueGetter(_dataContextType, PropertyName, _precedence, _allowPrivateMembers);
```

Comment:
This code is probably wrong. I think it should have been `BindingPropertyHelper.GetPrecedenceSpecificValueGetter` but I have no idea if that's the root cause.

Reply:
This is indeed interesting. Looking at https://github.com/unoplatform/uno/commit/5fe335c4d25aad59b3c6c76cbb77f945efe5fd5c, this was originally the case, and there this test:

https://github.com/unoplatform/uno/blob/ad7fe3d6e94eaee53242490230a9202efe4abb4c/src/Uno.UI.Tests/BinderTests/Given_Binder.cs#L616

was added at that time and that specifying a precedence to `GetValueGetter` was enough to make it precedence specific.